typescript/sdk/src/token/StarknetERC20WarpModule.ts

```diff
@@ -102,10 +102,9 @@ export class StarknetERC20WarpModule {
           addresses[chain] = tokenAddress;
           break;
         }
-
-        case TokenType.collateral: {
+        case TokenType.collateralDex: {
           const tokenAddress = await deployer.deployContract(
-            StarknetContractName.HYP_ERC20_COLLATERAL,
+            StarknetContractName.HYP_ERC20_DEX_COLLATERAL,
```

Comment:
Shouldn't this deploy the contract based on which token type we are supplying here? wouldn't this cause a dex collateral to be deployed on starknet even if we only want a collateral? are they equivalent ? sorry don't have lots of context on this

Reply:
TLDR on dexCollateral: it's a variant of the collateral token in cairo which deposits the tokens to paradex's DEX. So, it's only paradex specific.

Reply:
so yeah forcing to deploy dex collateral in any case would be wrong right?

Reply:
made the error of replacing `collateral` with `collateralDex`, just readded collateral and appended dexCollateral